packages/contracts-bedrock/test/L2/ETHLiquidity.t.sol

```diff
@@ -141,7 +141,7 @@ contract ETHLiquidity_Test is CommonTest {
     function testFuzz_mint_fromUnauthorizedCaller_fails(uint256 _amount, address _caller) public {
         // Assume
         vm.assume(_caller != address(superchainWeth));
-        vm.assume(address(_caller).balance == 0);
+        vm.deal(_caller, 0);
```

Comment:
I think this might break if the caller is the `ETHLiqudity` contract, might have to an assume there

Reply:
fixed